data/templates/ipsec/swanctl/remote_access.j2

```diff
@@ -34,6 +34,11 @@
 {% elif rw_conf.authentication.client_mode.startswith("eap") %}
             auth = {{ rw_conf.authentication.client_mode }}
             eap_id = %any
+{% endif %}
+{% if rw_conf.authentication.client_mode is vyos_defined('eap-tls') or rw_conf.authentication.client_mode is vyos_defined('x509') %}
+{#          in the case of an intermediate/root chain, assume user wants us to valiadate against #}
+{#          the lowest specified cert in the chain (e.g. the intermediate) #}
+            cacerts = {{ rw_conf.authentication.x509.ca_certificate }}_1.pem
```

Comment:
Once https://github.com/vyos/vyos-1x/pull/3202 is merged will need to change to:
`cacerts = {{ '.pem,'.join(rw_conf.authentication.x509.ca_certificate) ~ '.pem' }}`

(Should be correct, though will need testing)

Reply:
That syntax should work but I ended up taking what I thought was a "cleaner" approach of adding a filter. You are welcome to veto that though.